src/styles/components/drawerButtonStyles.ts

```diff
@@ -1,26 +1,13 @@
-const priceBarDrawer = {
-  variants: {
-    alwaysOpen: {
-      parts: ["dialog, dialogContainer"],
-      dialog: {
-        pointerEvents: "auto",
-      },
-      dialogContainer: {
-        pointerEvents: "none",
-      },
-    },
-  },
-};
-
 interface DrawerButtonProps {
   isOpen: boolean;
 }
 
-export const DrawerButton = {
+const drawerButtonStyles = {
   baseStyle: ({ isOpen }: DrawerButtonProps) => ({
```

Comment:
You can remove the interface and isOpen as these codes don't need anymore.

Reply:
Thanks Jin, fixed :)